elasticdl/python/common/lr_scheduler.py

```diff
@@ -0,0 +1,47 @@
+import threading
+
+
+class LearningRateScheduler:
+    def __init__(self, learning_rate=0.001, model_version=0):
```

Comment:
Do we need to support scheduling the learning rate by epoch?

Reply:
PS does not know how many batches in an epoch. The user knows it and can set the scheduler accordingly.